include/pgduckdb/scan/postgres_scan.hpp

```diff
@@ -39,14 +39,14 @@ class PostgresScanLocalState {
 	bool m_exhausted_scan;
 };
 
-struct PostgresReplacementScanData : public duckdb::ReplacementScanData {
+struct PostgresReplacementScanDataClientContextState : public duckdb::ClientContextState {
```

Comment:
Can we shorten this to `PostgresReplacementState`?

Also for future reference, I think a good portion of these class variables are unused, such as `m_rtables`